<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor</class>
 <widget class="QDialog" name="monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshProcessList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="processSelector"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="trackChanges">
       <property name="text">
        <string>Track changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearChanges">
       <property name="text">
        <string>Clear changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="onlyRealFiles">
       <property name="text">
        <string>Only real files</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="fileList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
